### DISCUSSION TITLE
Selecting all text copies line numbers

### DISCUSSION THREAD

**Discussion by MathiasKandelborg** [community/community#41993] [Projects and Issues]
### Select Topic Area

Bug

### Body

I'm trying to edit my ReadMe via a native editor. Trying to copy the markdown from GitHub proves to be difficult...

I'm using Firefox, if that means anything.

I'm getting **very** tired of all of these Microsoft issues...

**Comment by airtower-luna** [comment]
If you want to edit a file locally you don't need to copy text out of the browser. Instead you should clone the repository to your computer, edit the local file, commit, and then push. If those terms aren't familiar to you see the [Git Book website](https://git-scm.com/book/en/v2). Chapter 2 explains all you need for that, starting with [Getting a Git Repository](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository).

**Reply by MathiasKandelborg** [reply]
I'm a proficient programmer, who uses Git regularly. It seems you're not familiar with the ReadMe project, and misunderstood what I'm trying to do.

I want to copy the code in the browser editor, and paste it somewhere else. The problem is that I've usually been able to simply select the text in the HTML textbox, which is how HTML, and interacting with HTML elements work. For some reason, MS (GitHub but you know, MS can't stop touching something with this much data) has implemented some change to the editor, that makes it so one needs to get the raw file, instead of simply copying the code.

As a sidenote, try most online editors with line numbers, and see if you get the line numbers if you simply select multiple lines in the textbox. This is the first time I've experienced that from a website/browser editor of this magnitude.
